tests/fpsemi-examples.cpp

```diff
@@ -24,7 +24,7 @@
 #include <algorithm>  // for max, for_each
 #include <cstdint>    // for int64_t
 #include <cstdlib>    // for abs
-#include <numeric>    // for iota
+#include <numeric>  // for iota
```

Comment:
Please run `make format` and push again! I'll merge after that, nice one @MTWhyte